lib/index.ts

```diff
@@ -1,15 +1,15 @@
 // Copyright 2016 Google Inc. All Rights Reserved.
 // Licensed under the Apache License, Version 2.0. See LICENSE
 
-import {launch, LaunchedChrome} from 'chrome-launcher';
-const lighthouse = require('lighthouse');
+import { launch, LaunchedChrome } from 'chrome-launcher';
+const lighthouse = require('lighthouse/lighthouse-core');
 const parseChromeFlags = require('lighthouse/lighthouse-cli/run').parseChromeFlags;
-const perfConfig: any = require('./lh-config');
+const perfConfig: any = require('lighthouse/lighthouse-core/config/plots-config.js');
```

Comment:
I think you should just provide a config file here in this repo. We don't have much reason to maintain that plots-config fwiw. :)

### `metrics` audit
while everything you need is in the new audit called `metrics` it doesnt come with human-readable names.  Since you provide those in `messages.ts` maybe its fine and all you really need is `metrics`. I think this could work.


![image](https://user-images.githubusercontent.com/39191/40753281-2b44fa9c-6428-11e8-85b2-5e70d3ead629.png)


### individual audits
If you really need all the individual audits:

```js

module.exports = {
  extends: 'lighthouse:default',
  settings: {
    onlyAudits: defaultConfig.categories.performance.auditRefs
	.filter(a => a.group && a.group === 'metrics')
	.map(a => a.id)
  },
};


```

Reply:
Paul, thanks. Would you like to keep all metrics we already have or reduce their number? I'm kinda confused because I faced that people just ignore FCP and rely only on FMP in perf aduits/case studies etc. As result with 3.0 and plots-config I thought maybe trends were changed...

Reply:
The way we look at the problem now is:

### key metrics

* FCP
* Speed Index
* TTI

### worthwhile diagnostic metrics

* FMP
* FCPUIdle

EIL is interesting but could use some work. it'll also change substantially in the future. And it's not on the same timescale as the others.

FirstVC should mostly be duplicated by FCP.

VizComplete is cool unless it marked the wrong thing, in which case it and Speed Index are totally useless. :)

Reply:
@paulirish  but VizComplete is not part of metrics.
```js
auditRefs: [
        {id: 'first-contentful-paint', weight: 3, group: 'metrics'},
        {id: 'first-meaningful-paint', weight: 1, group: 'metrics'},
        {id: 'speed-index', weight: 4, group: 'metrics'},
        {id: 'interactive', weight: 5, group: 'metrics'},
        {id: 'first-cpu-idle', weight: 2, group: 'metrics'},
        {id: 'estimated-input-latency', weight: 0, group: 'metrics'},
```
Should it be added there?
Or,  should we use `pwmetrics-events` and get it manually?